test/sambah/SAMBAH_Regression.ipynb

```diff
@@ -0,0 +1,363 @@
+    "        'granule_id': [\n",
+    "            'G1263137623-LARC_CLOUD',\n",
+    "            'G1263137394-LARC_CLOUD',\n",
+    "            'G1263137387-LARC_CLOUD',\n",
+    "            'G1263137388-LARC_CLOUD',\n",
+    "            'G1263137378-LARC_CLOUD',\n",
+    "        ],\n",
+    "        # chosen variables include one variable from each group\n",
+    "        # support/scattering_weights is 3D variable\n",
+    "        'variables': [\n",
+    "            'product/vertical_column_stratosphere',\n",
+    "            'qa_statistics/fit_rms_residual',\n",
+    "            'support_data/scattering_weights',\n",
+    "        ],\n",
+    "    },\n",
+    "}\n",
+    "\n",
+    "req_info = req_infos.get(data_environment)\n",
+    "harmony_environment = host_environment.get(harmony_host_url)\n",
+    "\n",
+    "if harmony_environment is not None:\n",
+    "    harmony_client = Client(env=harmony_environment)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fc7aef85",
+   "metadata": {},
+   "source": [
+    "# Begin regression tests:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9dd58c71-0232-4a20-9dc4-ef6b3544e797",
+   "metadata": {},
+   "source": [
+    "## SAMBAH\n",
+    "\n",
+    "SAMBAH is currently deployed to Sandbox, SIT, UAT and production.\n",
+    "Requests will be made against the TEMPO NO2 L2 V03 collection."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ef9ec4d2-2228-47d0-a990-d5a4d77b7bc7",
+   "metadata": {},
+   "source": [
+    "### SAMBAH: temporal, variable and bounding box subset request\n",
+    "\n",
+    "This is a request that exercises the full range of SAMBAH options: temporal, bounding box and variable subsetting.\n",
+    "\n",
+    "Requested variable names:\n",
+    "\n",
+    "* `2024-3-28 15:34:00 < time < 2024-3-28 16:00:00`\n",
+    "* `product/vertical_column_total`\n",
+    "* `BBox(-170,30,-10,40)`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d6aaf063-1f3c-4d84-b414-ca80e65ef78b",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-05-17T17:55:05.307685Z",
+     "start_time": "2024-05-17T17:53:50.692944Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "temp_var_bbox_request = Request(\n",
+    "    collection=req_info['collection'],\n",
+    "    extend=['mirror_step'],\n",
+    "    granule_id=req_info['granule_id'],\n",
+    "    temporal=req_info['temporal'],\n",
+    "    variables=req_info['variables'],\n",
+    "    spatial=req_info['spatial'],\n",
+    ")\n",
+    "\n",
+    "\n",
+    "temp_var_bbox_filename = 'temp_var_bbox.nc4'\n",
+    "submit_and_download(harmony_client, temp_var_bbox_request, temp_var_bbox_filename)\n",
+    "assert exists(\n",
+    "    temp_var_bbox_filename\n",
+    "), 'Unsuccessful SAMBAH temporal, variable, bounding box request.'\n",
+    "\n",
+    "\n",
+    "compare_results_to_reference_file(temp_var_bbox_filename)\n",
```

Comment:
Could this be simplified by using datatree? (Albeit from [the old location](https://pypi.org/project/xarray-datatree/) prior to the next `xarray` release)

Reply:
using datatree now.